src/components/Sidebar/Sidebar.js

```diff
@@ -251,11 +251,11 @@ export class Sidebar extends PureComponent<Props, State> {
   }
 }
 
-const Wrapper = animated(styled.nav.attrs(props => ({
-  style: {
+const Wrapper = animated(styled.nav.attrs({
```

Comment:
Ups, merging master changed this to old styled-component syntax - I'll fix this.